### PR TITLE
feat: adds lw6 hackathon blog

### PR DESCRIPTION
**Pull request by awalias**

### CODE REVIEW

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+- Most Visually Pleasing (Swag kit)
+- Most Technically Impressive (Swag kit)
+- Most Fun/Interesting (Swag kit)
+- Best Storage Project (Swag kit)
+- Best Edge Functions Project (Deno Swag kit)
+
+Our friends at [Deno](https://deno.land/) have sponsored the Best Edge Functions Project category 🎉
+You can read more about the this sponsorship [here](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon)!
+
+There will be a winner and a runner-up prize for each category. Every winning team member gets a prize.
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:62]: @thorwebdev 
Would be great if you could take look!

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+![Gold cap](/images/blog/lw5-hackathon/cap.png)
+
+### Submission
+
+You should submit your project using [this form](https://www.madewithsupabase.com/launch-week-6).
+
+### Judges
+
+Best Edge Functions Project category will be judged by our friends at [Deno](https://deno.land/), and other categories will be judged by the Supabase team.
+We will be looking for:
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:74]: @thorwebdev Here as well!

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+![Supabase Keyboard](/images/blog/lw6-hackathon/keyboard.jpg)
+
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase.
+  - Starting 8:00am PT Friday 9th December 2022
+- Build whatever you want - a project, app, tool, library, anything
+- Enter as an individual, or as a team of up to 5 people
+- Submission deadline is 00:01am early Monday morning PT 19th December 2022
+- There are 5 prize categories.
```
Comment (thorwebdev) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:33]:
```suggestion
- There are 6 prize categories.
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.
+  - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!
```
Comment (thorwebdev) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:35]:
```suggestion
  - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!
- Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners! 
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+### Schedule
+
+- The Hackathon begins at 8:00am PT Friday 9th December 2022
+- Work on your project any time for the next 10 days
+- Submission deadline (00:01am early Monday morning PT 19th December 2022)
+
+### Prizes
+
+There are 5 chances to win, there will be prizes for:
```
Comment (thorwebdev) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:49]:
```suggestion
There are 6 chances to win, there will be prizes for:
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
+    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
+- Most Visually Pleasing (Swag kit)
+- Most Technically Impressive (Swag kit)
+- Most Fun/Interesting (Swag kit)
+- Best Storage Project (Swag kit)
+- Best Edge Functions Project (Deno Swag kit)
+
+Our friends at [Deno](https://deno.land/) have sponsored the Best Edge Functions Project category 🎉
```
Comment (thorwebdev) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:59]:
```suggestion
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+You can read more about the this sponsorship [here](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon)!
```
Comment (thorwebdev) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:60]:
```suggestion
```

**Review by thorwebdev — COMMENTED** [review]
@dshukertjr proposed changes. PTAL.

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+- Submission deadline (00:01am early Monday morning PT 19th December 2022)
+
+### Prizes
+
+There are 6 chances to win, there will be prizes for:
+
+- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
+    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
+- Most Visually Pleasing (Swag kit)
+- Most Technically Impressive (Swag kit)
```
Comment (awalias) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:55]:
```suggestion
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+### Schedule
+
+- The Hackathon begins at 8:00am PT Friday 9th December 2022
+- Work on your project any time for the next 10 days
+- Submission deadline (00:01am early Monday morning PT 19th December 2022)
+
+### Prizes
+
+There are 6 chances to win, there will be prizes for:
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:50]:
```suggestion
There are 5 chances to win, there will be prizes for:
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+![Supabase Keyboard](/images/blog/lw6-hackathon/keyboard.jpg)
+
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase.
+  - Starting 8:00am PT Friday 9th December 2022
+- Build whatever you want - a project, app, tool, library, anything
+- Enter as an individual, or as a team of up to 5 people
+- Submission deadline is 00:01am early Monday morning PT 19th December 2022
+- There are 6 prize categories.
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:33]:
```suggestion
- There are 5 prize categories.
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+![Gold cap](/images/blog/lw5-hackathon/cap.png)
+
+### Submission
+
+You should submit your project using [this form](https://www.madewithsupabase.com/launch-week-6).
+
+### Judges
+
+Best Edge Functions Project category will be judged by our friends at [Deno](https://deno.land/), and other categories will be judged by the Supabase team.
```
Comment (RamiroND) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:71]: Should we mention swyx?

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+[Join our Discord](https://discord.supabase.com)
+
+![Discord Hangout](/images/blog/hackathon/community.png)
+
+### Launch Week
+
+Don't forget to checkout the new feature's being announced as part of [Launch Week](/launch-week).
+
+- [Previous Hackathon Prize Winners](https://supabase.com/blog/launch-week-5-hackathon-winners)
+- [Open Source at Supabase - Founders Fireside Chat](https://www.youtube.com/watch?v=4t_63HT3rZY)
```
Comment (awalias) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:108]:
```suggestion
- [Who We Hire at Supabase - Founders Chat](https://youtu.be/-BG9XptyCKI)
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+
+- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
+    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
+- Most Visually Pleasing (Swag kit)
+- Most Fun/Interesting (Swag kit)
+- Best Storage Project (Swag kit)
+- Best Edge Functions Project (Deno Swag kit)
+
+
+There will be a winner and a runner-up prize for each category. Every winning team member gets a prize.
```
Comment (awalias) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:57]:
```suggestion
There will be winner and runner-up prizes for each category. Every team member on winning/runner-up teams gets a supaverified swag kit.
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+- Submission deadline (00:01am early Monday morning PT 19th December 2022)
+
+### Prizes
+
+There are 5 chances to win, there will be prizes for:
+
+- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
+    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
+- Most Visually Pleasing (Swag kit)
+- Most Fun/Interesting (Swag kit)
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:52]:
```suggestion
- Most Fun/Interesting (Swag kit)
- Best Flutter Project (Swag kit)
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase.
+  - Starting 8:00am PT Friday 9th December 2022
+- Build whatever you want - a project, app, tool, library, anything
+- Enter as an individual, or as a team of up to 5 people
+- Submission deadline is 00:01am early Monday morning PT 19th December 2022
+- There are 5 prize categories.
+  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.
+  - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:34]:
```suggestion
  - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+
+- You have 10 days to build a new **Open Source** project using Supabase.
+  - Starting 8:00am PT Friday 9th December 2022
+- Build whatever you want - a project, app, tool, library, anything
+- Enter as an individual, or as a team of up to 5 people
+- Submission deadline is 00:01am early Monday morning PT 19th December 2022
+- There are 5 prize categories.
+  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.
+  - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
+- Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners!
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:35]:
```suggestion
  - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
  - Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners! 
```

File: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase.
+  - Starting 8:00am PT Friday 9th December 2022
+- Build whatever you want - a project, app, tool, library, anything
+- Enter as an individual, or as a team of up to 5 people
+- Submission deadline is 00:01am early Monday morning PT 19th December 2022
+- There are 5 prize categories.
+  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.
+  - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
```
Comment (dshukertjr) [inline apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx:34]:
```suggestion
  - Most fun/interesting, best Flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
```

**Review by dshukertjr — APPROVED** [review]
I think it looks good!

**Pull request merged**